app/routes.py

```diff
@@ -1,2 +1,51 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response
 
+
+class Planet:
+    def __init__(self, id, name, description, association):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.association = association
+
+    def make_planet_dict(self):
+        return dict(
+            id=self.id,
+            name=self.name,
+            description=self.description,
+            association=self.association)
+
+
+planets = [
+    Planet(1, "Mercury", "smallest planet", "Wednesday"),
+    Planet(2, "Mars", "spicy, red one", "Tuesday"),
+    Planet(3, "Jupiter", "biggest planet", "Thursday"),
+    Planet(4, "Venus", "hottest planet", "Friday"),
+    Planet(5, "Saturn", "the one with the rings", "Saturday")
+]
+
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
```

Comment:
✅ Blueprint looks good. Though typically when there's only a single Blueprint in a file (often the way we set things up), we'll call the variable holding the blueprint just `bp`.